changelog/2560.doc.rst

```diff
@@ -0,0 +1 @@
+Updated the docstring for plasmapy.formulary.lengths.gyroradius to finish an unfinished sentence.
```

Comment:
```suggestion
Updated the docstring for `~plasmapy.formulary.lengths.gyroradius to` finish an unfinished sentence.
```
The backticks will make a link to the documentation for `gyroradius`, and the tilde indicates that the namespace should be hidden.